tests/bluetooth/tester/src/gatt.c

```diff
@@ -2029,7 +2029,10 @@ void tester_handle_gatt(uint8_t opcode, uint8_t index, uint8_t *data,
 		read_long(data, len);
 		return;
 	case GATT_READ_MULTIPLE:
-		read_multiple(data, len);
+		read_multiple(data, len, false);
+		return;
+	case GATT_READ_MULTIPLE_VAR:
```

Comment:
You need to add this command into s[upported command](https://github.com/zephyrproject-rtos/zephyr/blob/d3094322d884ab64f6609b846e93d1684b7bab0a/tests/bluetooth/tester/src/gatt.c#L205) also.

Reply:
Done